src/devices/namron.ts

```diff
@@ -884,6 +884,49 @@ const definitions: Definition[] = [
         toZigbee: [],
         exposes: [e.battery(), e.temperature(), e.humidity()],
     },
+    {
+        zigbeeModel: ['4512766', '4512767'],
+        model: '4512766 / 4512767',
+        vendor: 'NAMRON',
+        description: 'Namron Zigbee smart plug',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.ignore_basic_report, fz.device_temperature],
+        toZigbee: [tz.on_off],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'haElectricalMeasurement', 'seMetering', 'genDeviceTempCfg']);
+            endpoint.saveClusterAttributeKeyValue('seMetering', {divisor: 100000, multiplier: 1});
+            endpoint.saveClusterAttributeKeyValue('haElectricalMeasurement', {
+                acVoltageMultiplier: 1, acVoltageDivisor: 1, acCurrentMultiplier: 1, acCurrentDivisor: 1000, acPowerMultiplier: 1,
+                acPowerDivisor: 1,
+            });
+            try {
+                await reporting.deviceTemperature(endpoint);
+                await reporting.currentSummDelivered(endpoint);
+                await reporting.rmsVoltage(endpoint, {change: 5});
+                await reporting.rmsCurrent(endpoint, {change: 50});
+                await reporting.activePower(endpoint, {change: 10});
+            await endpoint.read('genOnOff', ['onOff']);
+        
```

Comment:
Could you fix this syntax error?

Reply:
Keep getting errors, closing pull to reopen when i got this figured out